app/src/main/res/values-nl/strings.xml

```diff
@@ -0,0 +1,259 @@
+    <string name="empty_excluded_dirs_detail">Jockey kan beperkt worden om sommige mappen over te slaan, druk op de “+” knop om een map uit te sluiten.</string>
+    <string name="url_privacy_policy">https://www.marverenic.com/Jockey/privacy-policy.html</string>
+    <string name="first_launch_detail">"<![CDATA[Door deze applicatie te gebruiken ga je akkoord met Jockey\'s         <a href="https://www.marverenic.com/Jockey/privacy-policy.html">privacybeleid</a>.]]>     "</string>
+    <string name="pref_resume_on_connect">Afspelen als headset verbonden</string>
+    <string name="pref_resume_on_connect_detail_on">Muziek zal spelen als een hoofdtelefoon verbonden wordt en Jockey aan staat.</string>
+    <string name="pref_resume_on_connect_detail_off">Muziek zal nooit automatisch spelen bij het verbinden van de hoofdtelefoon</string>
+    <string name="gesture_detail">Veeg over de albumillustratie om naar het volgende nummer te gaan, of raak aan om te pauzeren</string>
+    <string name="gesture_description">Gebaren inschakelen</string>
+    <string name="pref_sls_action_install">Installeer Simple Last.fm Scrobbler</string>
+    <string name="pref_sls_action_open">Open Simple Last.fm Scrobbler instellingen</string>
+    <string name="pref_sls_scrobble">Scrobble met Simple Last.fm Scrobbler</string>
+    <string name="pref_sls_description">Jockey kan samenwerken met Simple Last.fm Scrobbler om je afspeelhistorie online te delen. De volgende keer dat je een nummer afspeelt zal dit online worden gezet.</string>
+    <string name="pref_sls_description_not_installed">"<![CDATA[         Jockey kan je afspeelhistorie online plaatsen naar services zoals Last.fm door te integreren met         een onafhankelijke applicatie. Download Simple Last.fm Scrobbler van de         Google Play Store.         <br/><br/>         Je kan meer informatie vinden over Simple Last.fm Scrobbler vinden        <a href="https://github.com/tgwizard/sls/">op zijn website</a>.]]>     "</string>
+    <string name="pref_sls_description_enabled_but_uninstalled">"<![CDATA[        Om verder te gaan met scrobbelen naar Last.fm, Simple Last.fm Scrobbler moet geinstalleerd blijven. Je         afspeelhistorie zal dan niet worden gedeeld. Als je niet langer wil scrobbelen kan je dit uitzetten.         <br/><br/>         Jockey kan je afspeelhistorie online plaatsen naar services zoals Last.fm door te integreren met         een onafhankelijke applicatie. Download Simple Last.fm Scrobbler van de         Google Play Store.         <br/><br/>         Je kan meer informatie vinden over Simple Last.fm Scrobbler vinden        <a href="https://github.com/tgwizard/sls/">op zijn website</a>.]]>     "</string>
+    <string name="equalizer_conflict_note">Er is een andere equalizer app geinstalleerd op dit apparaat. Dit kan problemen veroorzaken met Jockey\'s equalizer. Schakel Jockey\'s equalizer uit om de systeem-wijde equalizer te gebruiken.</string>
+</resources>
```

Comment:
I just remembered there's one more thing this file will need, and it's really subtle (and quite frankly, I have no idea how to even begin researching this aside from talking to someone who knows the language).

In English, when sorting music, books, and other works based on their titles, we ignore the articles "The", "A", and "An" when they appear at the start of the title. So instead of placing "The Bridge Between" in the same place as songs with titles that begin with 'T', we place them in the same place as songs with titles that start with the letter 'B'. Does Dutch have a similar convention? If it does, can you add this array to the end of this file?

```xml
    <!-- Prefixes that are ignored when sorting titles -->
    <string-array name="ignored_title_prefixes" tools:ignore="InconsistentArrays">
        <item>The</item>
        <item>A</item>
        <item>An</item>
    </string-array>
```

If there is no such convention, you can just define an empty `<string-array ...>` with no `<item>`'s which will disable this sorting behavior (Currently, it will fall back and ignore English articles if they appear at the beginning of a title, which might not be the right thing to do).

Reply:
There is no national standard for music, however from my experience with keeping a vinyl/CD collection I can say that most people ignore the prefixes.

I've added the piece of code to the end of the file and committed it, but Android Studio displays an error: "Namespace 'tools' is not bound". Not sure if that's just my configuration or if it's incorrect.

---

Also, when sorting music by the *artist*, we ignore the following prefixes (capitalisation ignored):
```xml
<string-array name="ignored_title_prefixes" tools:ignore="InconsistentArrays">
        <item>Van</item>
        <item>De</item>
        <item>In</item>
        <item>Ten</item>
        <item>Van den</item>
        <item>Van der</item>
</string-array>
```
and surnames starting with `IJ` are sorted under the `Y`

---

References: [1](https://sec-florijn.learningmatters.nl/informatiemanagement/p1/alfabetisch-rangschikken), [2](https://nl.wikipedia.org/wiki/Alfabetische_volgorde)

Reply:
Oh. That's awesome to know. There isn't a way to designate that logic right now (since English doesn't have similar rules), but that's definitely something that can be built in with this knowledge.

That said, the surname rule will be virtually impossible to code in since there isn't a good way to figure out whether a word is a surname or not. Let me know if there's a heuristic that might work as an approximation for that rule, or if we should just ignore it for now.

(Also, GitHub is really weird about replying in threads during a review, so this comment will appear as part of the review, but to reply to it you have to scroll back up to the original thread. No idea why they did it that way)